src/DataClasses/link.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+class Link():
+	'''
+	Constructor
+	Parameters: owner (owner object), properties (list of properties associated with that owner)
+	'''
+	def __init__(self, owner, properties):
+		self.owner = owner
```

Comment:
This is interesting that you made the owner the "top level entity" and that multiple addresses are associated with one owner.  This is definitely true in reality.  But there are also properties that are owned by many owners.  I was thinking that we would be targeting properties, as in the property would be our starting point, so I was envisioning an address with multiple owners.  Maybe we need to consider modeling a many to many relationship.